radix-engine/src/utils/native_blueprint_call_validator.rs

```diff
@@ -15,69 +14,79 @@ use radix_engine_interface::blueprints::resource::*;
 use radix_engine_interface::object_modules::metadata::*;
 use radix_engine_interface::object_modules::role_assignment::*;
 use radix_engine_interface::object_modules::royalty::*;
+use radix_transactions::manifest::*;
 use radix_transactions::prelude::*;
 
-pub fn validate_call_arguments_to_native_components(
-    instructions: &[impl ManifestInstruction],
+pub fn validate_call_arguments_to_native_components_any(
+    manifest: &AnyTransactionManifest,
```

Comment:
Ahh with a later PR, it could take `&[impl InstructionSet]` - happy to change it, but I think this is also OK as is.
A manifest allows you to give better error messages, as you can e.g. resolve names for buckets.